test/run.js

```diff
@@ -0,0 +1,20 @@
+const spawn = require('child_process').spawn
+
+const {start, stop} = require('./server')
+
+start(() => {
```

Comment:
I think it would be simpler & more flexible to require the testrpc chain to be running already. In the `.travis.yml` we might use something like this:

```
before_script:
  - node test/chain.js
script:
  - npm test
```

Doing that, we don't have to spawn subprocesses.